ow_plexil/src/plans/FaultHandlingPattern3.plp

```diff
@@ -37,20 +46,29 @@ FaultHandlingPattern3: UncheckedSequence
   {
     Start !Lookup(ArmFault);
     Invariant !Lookup(ArmFault);
-    Post Lookup(ActionGoalStatus("ArmUnstow")) == ACTION_SUCCEEDED;
 
     LibraryCall ArmUnstow();
   }
 
   UnstowSuccess = Unstow.outcome == SUCCESS;
 
-  Stow:
+  Stow: Concurrence
   {
+    Boolean FaultOccurred = false;
     Start !Lookup(ArmFault);
-    Invariant !Lookup(ArmFault);
-    Post Lookup(ActionGoalStatus("ArmStow")) == ACTION_SUCCEEDED;
+    Post !FaultOccurred;
```

Comment:
This seems like it should work, but I am wondering if the problem I saw in my test will be resolved by this change?
```suggestion
    Post TheStow.outcome == SUCCESS;
```

Reply:
Your suggested change won't work because `TheStow` always succeeds.  The only way a node can fail is having a check condition (Pre, Post, Invariant) that fails.

These reports are instances of the caveat I mention in the plan comments.  I'd prefer to work on this bug via the new Jira I created for it.

Reply:
> Your suggested change won't work because `TheStow` always succeeds.

If that is the case then how is it that I see `false` for `UnstowSuccess`, which does what I suggested but without using a PostCondition? My understanding of PLEXIL was that so long as a Node only has a command call that Node becomes a Command Node, and so long as the proper API was hooked up for that command to signal failure, a Command Node will have an `outcome` of `FAILURE` if the command it calls fails. Does this sound correct?

Thanks for clarifying these are part of your caveat. I have no issue with this bug being addressed in a separate ticket.

Reply:
The Unstow node can fail through its Invariant condition, and no other way.  A Command node's success is actually agnostic of whether the command succeeds or not, unless the node has an explicit check condition (pre, post, invariant) on the command handle.